hw/xbox/xbox.c

```diff
@@ -227,11 +227,62 @@ static void xbox_memory_init(PCMachineState *pcms,
     xbox_flash_init(rom_memory);
 }
 
+uint8_t* load_eeprom(void)
```

Comment:
Nit: `*` goes to the immediate left of the identifier (e.g. `uint8_t *load_eeprom`). Repeated a couple times below.